pkgs/trebuchet/README.md

```diff
@@ -0,0 +1,7 @@
+Run
+
+```bash
+dart run bin/trebuchet.dart --input-name coverage --branch-name master --input-path ~/projects/coverage/ --target-path ~/projects/tools/ --git-filter-repo ~/tools/git-filter-repo 
```

Comment:
Perhaps use `\` continuation lines to split this longer example up into a few lines?